lib/parser/onnx/onnx_parser.cc

```diff
@@ -676,4 +711,75 @@ std::unique_ptr<Parser> CreateONNXParser() {
   return std::make_unique<ONNXParser>();
 }
 
-} // end namespace halo
\ No newline at end of file
+Status ONNXParser::ConvertLoopNode(IRBuilder* ir_builder,
+                                   const onnx::NodeProto& cur_node) {
+  std::string cur_node_name = cur_node.name();
+  if (cur_node_name.empty()) {
+    // TODO(unknown) current node name must unique
+    cur_node_name = "unknown";
+  }
+  const auto& operands = GetInputOperands(cur_node);
+  // first 2 inputs(loop_cnt/loop_cond) is optional
+  for (int i = 0, n = operands.size(); i < n; ++i) {
+    loop_arg_types_.push(operands[i].GetType());
```

Comment:
1. why use stack here? The order will be reversed when you use it in L437.

Reply:
you are right, the order is wrong, i will reverse it when